src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -1028,6 +1028,9 @@ export const GlobalProvider = memo(
                 });
                 outputDataActions.delete(id);
                 addInputDataChanges();
+                backend.clearNodeCacheIndividual(id).catch((error) => {
+                    log.error(error);
+                });
```

Comment:
Super minor nit: Could we make this a single line?

```suggestion
                backend.clearNodeCacheIndividual(id).catch((error) => log.error(error));
```